libcontainer/container_linux.go

```diff
@@ -195,12 +191,23 @@ func (c *linuxContainer) Run(process *Process) error {
 	if err != nil {
 		return err
 	}
-	isInit := status == Stopped
-	if err := c.start(process, isInit); err != nil {
+	if err := c.start(process, status == Stopped); err != nil {
 		return err
 	}
-	if isInit {
-		return process.ops.signal(InitContinueSignal)
+	if status == Stopped {
+		return c.Exec()
+	}
+	return nil
+}
+
+func (c *linuxContainer) Exec() error {
```

Comment:
On Mon, Jun 06, 2016 at 05:06:57PM -0700, Michael Crosby wrote:
> > +func (c *linuxContainer) Exec() error {
> 
> its not changing because we already have a start internally that is
> used in other places

Ah, in that case the missmatch with the spec is probably unavoidable.